packages/project/src/index.ts

```diff
@@ -0,0 +1,7 @@
+export * as project from "./project";
+export * as breakpoints from "./breakpoints";
+export * as tree from "./tree";
+export * as props from "./props";
+
+export * from "./shared/breakpoints";
+export * from "./shared/tree-utils";
```

Comment:
ok, here is the suggestion, lets export it something like this:

```suggestion
import * as project from "./project";
import * as breakpoints from "./breakpoints";
import * as tree from "./tree";
import * as props from "./props";

export const db = {project, breakpoints, tree, props}

export breakpoints from "./shared/breakpoints";
export tree from "./shared/tree-utils";
```